docker/example.env

```diff
@@ -0,0 +1,16 @@
+MF_USERS_LOG_LEVEL="info"
+MF_JAEGER_URL="localhost:6831"
+MF_GCP_CRED="creds.json"
+MF_SPREADSHEET_ID=''
+MF_SHEET_ID=0
+MF_IP_DB="IP2LOCATION-LITE-DB5.BIN"
+MF_AUTH_GRPC_PORT=7001
+MF_AUTH_GRPC_URL=mainflux-auth:7001
+MF_AUTH_GRPC_TIMEOUT=1s
+MF_TIMESCALE_HOST="timescaledb"
+MF_TIMESCALE_PORT=5432
+MF_TIMESCALE_USER="mainflux"
+MF_TIMESCALE_PASSWORD="mainflux"
+MF_TIMESCALE_DB_NAME="mainflux"
+ET_RELEASE_TAG="latest"
+ET_PORT=8855
```

Comment:
Add empty line

also rename it to .env

Reply:
my actual .env contains secrets I can't commit on github